test/studies/comd/elegant/arrayOfStructs/util/EAM.chpl

```diff
@@ -54,8 +54,8 @@ class PotentialEAM : BasePotential {
   var phiIO, rhoIO, fIO : unmanaged InterpolationObject;
 
   var rhoDom : domain(3);
-  var rhoSpace = rhoDom dmapped AccumStencil(rhoDom); // basically BlockDist
-  var dfSpace  = rhoDom dmapped AccumStencil(rhoDom, fluff=(1,1,1), periodic=true);
+  var rhoSpace = rhoDom dmapped new dmap(new AccumStencil(rhoDom)); // basically BlockDist
+  var dfSpace  = rhoDom dmapped new dmap(new AccumStencil(rhoDom, fluff=(1,1,1), periodic=true));
```

Comment:
Do you need the wrapper `dmap` record here? The syntax described above, even in the deprecation wording, seems to be `dmapped new <DistName>` just from the error above. If you do, then I think you need to adjust the error message to explain when this is needed.